src/NuGet.Core/NuGet.Frameworks/FrameworkException.cs

```diff
@@ -14,6 +14,9 @@ public FrameworkException(string message)
         {
         }
 
+#if NET8_0_OR_GREATER
+        [Obsolete(DiagnosticId = "SYSLIB0051")] // add this attribute to the serialization ctor
```

Comment:
Consider replaceing the `// add this attribute to the serialization ctor` comment with a reference to the breaking change that remove the StreamingContext overload.

Reply:
Absolutely - this was a simple copy from the guidance doc - will update.